PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Infrastructure/Migrations/20241113163025_AddUserTypeDiscriminator2.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+#pragma warning disable CA1814 // Prefer jagged arrays over multidimensional
```

Comment:
REMOVE ALL MIGRATIONS AND LEAVE ONLY RELEVANT ONE IN PR. Your migrations now look very messy